packages/cfd/src/Containers/__tests__/cfd-dashboard-container.spec.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { render, screen, fire, fireEvent } from '@testing-library/react';
+import CFDDashboardContainer from '../cfd-dashboard-container';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Icon: jest.fn(props => <div>{props.icon}</div>),
+    };
+});
+
+describe('CFDDashboardContainer', () => {
+    const mock_props = {
+        platform: 'mt5',
+        active_index: 0,
+        is_dark_mode_on: false,
+    };
+
+    it('should render <CFDDashboardContainer /> correctly', () => {
+        render(<CFDDashboardContainer {...mock_props} />);
+        expect(screen.getByTestId(/dt_cfd_dashboard_download_center_container/i)).toBeInTheDocument();
+    });
+    it('should render correct text according to the MT5 platform', () => {
+        render(<CFDDashboardContainer {...mock_props} />);
+        expect(
+            screen.getByText(/Run MT5 from your browser or download the MT5 app for your devices/i)
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(/The MT5 desktop app is not supported by Windows XP, Windows 2003, and Windows Vista/i)
+        ).toBeInTheDocument();
+        expect(screen.getByTestId(/dt_mt5_text/i)).toBeInTheDocument();
```

Comment:
Here, we are testing the actual text in line 29 and  testing the same with testid in line 32.  Do we need both?

Reply:
Nope. will remove the testid